2013/05/03 14:07:43 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\Projekt-Zespolowy\protected\models\User.php(28): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\Projekt-Zespolowy\protected\components\UserIdentity.php(13): User::model()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#10 C:\wamp\www\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#11 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#12 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#13 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(105): CModel->validate()
#14 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#15 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/login
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/login
---
2013/05/03 14:08:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 14:08:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 14:08:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 14:09:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 14:09:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 14:33:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Article))
#7 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(402): CActiveRecord::model('Article')
#8 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(77): CActiveRecord->getMetaData()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(93): CActiveRecord->__construct()
#10 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionArticle()
#11 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('article')
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/article')
#16 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/article
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:13:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "article".' in C:\wamp\www\Yii\framework\web\CController.php:483
Stack trace:
#0 C:\wamp\www\Yii\framework\web\CController.php(270): CController->missingAction('article')
#1 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('article')
#2 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/article')
#3 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/article
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:14:03 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (97)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:14:03 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:15:56 [error] [exception.CException] exception 'CException' with message 'Property "SubmitArticleForm.file" is not defined.' in C:\wamp\www\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\wamp\www\Yii\framework\web\helpers\CHtml.php(2255): CComponent->__get('file')
#1 C:\wamp\www\Yii\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(SubmitArticleForm), 'file')
#2 C:\wamp\www\Yii\framework\web\helpers\CHtml.php(1257): CHtml::activeInputField('text', Object(SubmitArticleForm), 'file', Array)
#3 C:\wamp\www\Yii\framework\web\widgets\CActiveForm.php(657): CHtml::activeTextField(Object(SubmitArticleForm), 'file', Array)
#4 C:\wamp\www\Yii\Projekt-Zespolowy\protected\views\site\submitArticle.php(69): CActiveForm->textField(Object(SubmitArticleForm), 'file')
#5 C:\wamp\www\Yii\framework\web\CBaseController.php(126): require('C:\wamp\www\Yii...')
#6 C:\wamp\www\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\wamp\www\Yii...', Array, true)
#7 C:\wamp\www\Yii\framework\web\CController.php(869): CBaseController->renderFile('C:\wamp\www\Yii...', Array, true)
#8 C:\wamp\www\Yii\framework\web\CController.php(782): CController->renderPartial('submitArticle', Array, true)
#9 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(107): CController->render('submitArticle', Array)
#10 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#11 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#16 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:20:35 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (97)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:20:35 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:20:53 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (97)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:20:53 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:21:02 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (97)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:21:02 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:21:27 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (97)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:21:27 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:21:28 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (97)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:21:28 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(99): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:22:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:22:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:22:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:22:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\Yii\framework\validators\CUniqueValidator.php(87): CActiveRecord::model('User')
#8 C:\wamp\www\Yii\framework\validators\CValidator.php(213): CUniqueValidator->validateAttribute(Object(SubmitRegisterForm), 'username')
#9 C:\wamp\www\Yii\framework\base\CModel.php(159): CValidator->validate(Object(SubmitRegisterForm), NULL)
#10 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(84): CModel->validate()
#11 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitRegister()
#12 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#16 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#17 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#18 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:34:00 [error] [exception.CException] exception 'CException' with message 'Property "SubmitArticleForm.isNewRecord" is not defined.' in C:\wamp\www\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\views\site\submitArticle.php(50): CComponent->__get('isNewRecord')
#1 C:\wamp\www\Yii\framework\web\CBaseController.php(126): require('C:\wamp\www\Yii...')
#2 C:\wamp\www\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\wamp\www\Yii...', Array, true)
#3 C:\wamp\www\Yii\framework\web\CController.php(869): CBaseController->renderFile('C:\wamp\www\Yii...', Array, true)
#4 C:\wamp\www\Yii\framework\web\CController.php(782): CController->renderPartial('submitArticle', Array, true)
#5 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(107): CController->render('submitArticle', Array)
#6 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#7 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#11 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#12 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:39:47 [error] [exception.CException] exception 'CException' with message 'Property "SubmitArticleForm.isNewRecord" is not defined.' in C:\wamp\www\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\views\site\submitArticle.php(50): CComponent->__get('isNewRecord')
#1 C:\wamp\www\Yii\framework\web\CBaseController.php(126): require('C:\wamp\www\Yii...')
#2 C:\wamp\www\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\wamp\www\Yii...', Array, true)
#3 C:\wamp\www\Yii\framework\web\CController.php(869): CBaseController->renderFile('C:\wamp\www\Yii...', Array, true)
#4 C:\wamp\www\Yii\framework\web\CController.php(782): CController->renderPartial('submitArticle', Array, true)
#5 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(111): CController->render('submitArticle', Array)
#6 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#7 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#11 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#12 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:43:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Nie mo¿na nawi¹zaæ po³¹czenia, poniewa¿ komputer docelowy aktywnie go odmawia.
' in C:\wamp\www\Yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\wamp\www\Yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\wamp\www\Yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\wamp\www\Yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\wamp\www\Yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(SubmitArticleForm))
#7 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(402): CActiveRecord::model('SubmitArticleFo...')
#8 C:\wamp\www\Yii\framework\db\ar\CActiveRecord.php(77): CActiveRecord->getMetaData()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(93): CActiveRecord->__construct()
#10 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#11 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#15 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#16 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitRegister
---
2013/05/03 15:48:10 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (102)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:48:10 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:48:12 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (102)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:48:12 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:48:16 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (102)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:48:16 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:48:17 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (102)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:48:17 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:48:23 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (102)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:48:23 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/03 15:51:43 [warning] [application] Failed to set unsafe attribute "files" of "SubmitArticleForm".
in C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php (102)
in C:\wamp\www\Yii\Projekt-Zespolowy\index.php (13)
2013/05/03 15:51:43 [error] [exception.CException] exception 'CException' with message 'SubmitArticleForm and its behaviors do not have a method or closure named "save".' in C:\wamp\www\Yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): CComponent->__call('save', Array)
#1 C:\wamp\www\Yii\Projekt-Zespolowy\protected\controllers\SiteController.php(104): SubmitArticleForm->save()
#2 C:\wamp\www\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionSubmitArticle()
#3 C:\wamp\www\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp\www\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp\www\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp\www\Yii\framework\web\CWebApplication.php(282): CController->run('submitArticle')
#7 C:\wamp\www\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitArti...')
#8 C:\wamp\www\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\wamp\www\Yii\Projekt-Zespolowy\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
HTTP_REFERER=http://localhost/Yii/Projekt-Zespolowy/index.php?r=site/submitArticle
---
2013/05/06 10:31:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submitRegister".' in C:\xampp\htdocs\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\framework\web\CController.php(270): CController->missingAction('submitRegister')
#1 C:\xampp\htdocs\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#2 C:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#3 C:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\informatica\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/informatica/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/informatica/
---
2013/05/06 10:31:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submitRegister".' in C:\xampp\htdocs\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\framework\web\CController.php(270): CController->missingAction('submitRegister')
#1 C:\xampp\htdocs\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#2 C:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#3 C:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\informatica\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/informatica/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/informatica/index.php?r=site/submitRegister
---
2013/05/06 10:32:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submitRegister".' in C:\xampp\htdocs\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\framework\web\CController.php(270): CController->missingAction('submitRegister')
#1 C:\xampp\htdocs\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#2 C:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#3 C:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\informatica\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/informatica/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/informatica/index.php?r=site/submitRegister
---
2013/05/06 10:32:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "submitRegister".' in C:\xampp\htdocs\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\framework\web\CController.php(270): CController->missingAction('submitRegister')
#1 C:\xampp\htdocs\framework\web\CWebApplication.php(282): CController->run('submitRegister')
#2 C:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController('site/submitRegi...')
#3 C:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\informatica\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/informatica/index.php?r=site/submitRegister
HTTP_REFERER=http://localhost/informatica/index.php?r=site/submitRegister
---
